IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -56,7 +57,16 @@ void HandleApplicationIdle(object sender, EventArgs e)
                     {
                         // Close the form after the test completes. This will unblock the 'showForm()' call if it's
                         // waiting for the form to close.
-                        form.Close();
+                        try
```

Comment:
❕ The try/finally block here is unnecessary. The `JoinPendingOperationsAsync` only exists to ensure successful cleanup after successful close. If the close throws an exception, the test has already failed and we're still going to join the operations here:
https://github.com/gitextensions/gitextensions/blob/147eb46ff0f0caa5d80ef46c772b0bf3f7ba5605/UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs#L74

Reply:
You have convinced me. If the close fails, the test will hang most likely anyway. Then it does not matter whether the async operations are tried to join during `Application.Run(form)` or afterwards.